core/data/src/test/java/uk/gov/gchq/gaffer/data/elementdefinition/view/ViewTest.java

```diff
@@ -211,154 +212,118 @@ public void shouldJsonSerialiseAndDeserialise() {
-                "      \"groupBy\" : [ \"property1\" ],%n" +
-                "      \"transientProperties\" : {%n" +
-                "        \"property2\" : \"java.lang.String\"%n" +
-                "      },%n" +
-                "      \"preAggregationFilterFunctions\" : [ {%n" +
-                "        \"predicate\" : {%n" +
-                "          \"class\" : \"uk.gov.gchq.gaffer.function.ExampleFilterFunction\"%n" +
-                "        },%n" +
-                "        \"selection\" : [ \"property1\" ]%n" +
-                "      } ],%n" +
-                "      \"postAggregationFilterFunctions\" : [ {%n" +
-                "        \"predicate\" : {%n" +
-                "          \"class\" : \"uk.gov.gchq.gaffer.function.ExampleFilterFunction\"%n" +
-                "        },%n" +
-                "        \"selection\" : [ \"VERTEX\" ]%n" +
-                "      } ]%n" +
-                "    },%n" +
-                "    \"BasicEntity\" : {%n" +
-                "      \"groupBy\" : [ \"property1\" ],%n" +
-                "      \"transientProperties\" : {%n" +
-                "        \"property2\" : \"java.lang.String\"%n" +
-                "      },%n" +
-                "      \"preAggregationFilterFunctions\" : [ {%n" +
-                "        \"predicate\" : {%n" +
-                "          \"class\" : \"uk.gov.gchq.gaffer.function.ExampleFilterFunction\"%n" +
-                "        },%n" +
-                "        \"selection\" : [ \"property1\" ]%n" +
-                "      } ],%n" +
-                "      \"postAggregationFilterFunctions\" : [ {%n" +
-                "        \"predicate\" : {%n" +
-                "          \"class\" : \"uk.gov.gchq.gaffer.function.ExampleFilterFunction\"%n" +
-                "        },%n" +
-                "        \"selection\" : [ \"VERTEX\" ]%n" +
-                "      } ]%n" +
-                "    },%n" +
-                "    \"BasicEntity3\" : {%n" +
-                "      \"groupBy\" : [ \"dateProperty\" ],%n" +
-                "      \"transientProperties\" : {%n" +
-                "        \"property2\" : \"java.lang.String\"%n" +
-                "      },%n" +
-                "      \"preAggregationFilterFunctions\" : [ {%n" +
-                "        \"predicate\" : {%n" +
-                "          \"class\" : \"uk.gov.gchq.gaffer.function.ExampleFilterFunction\"%n" +
-                "        },%n" +
-                "        \"selection\" : [ \"property1\" ]%n" +
-                "      }, {%n" +
-                "        \"predicate\" : {%n" +
-                "          \"class\" : \"uk.gov.gchq.gaffer.function.ExampleFilterFunction\"%n" +
-                "        },%n" +
-                "        \"selection\" : [ \"dateProperty\" ]%n" +
-                "      } ]%n" +
-                "    }%n" +
-                "  }%n" +
-                "}"), new String(view.toJson(true)));
+            "  \"edges\" : {%n" +
```

Comment:
Please can you tell your IDE to not change this indenting? Also occurred in the SchemaTest file.